src/ngAnimate/.eslintrc.json

```diff
@@ -1,7 +1,9 @@
 {
-  "extends": "../../.jshintrc-base",
-  "maxlen": false, /* ngAnimate docs contain wide tables */
-  "newcap": false,
+  "rules": {
+    "maxlen": "off", /* ngAnimate docs contain wide tables */
+    "new-cap": "off",
+  },
+
```

Comment:
Yup, configs are merged up the tree up until "/" (or "C:\" or whatever you
have in Windows-land ;)) unless one of those configs has `"root": true` -
then the lookup ends on that file.